data/forms/warning-note-review.tsx

```diff
@@ -59,7 +59,7 @@ const formSteps: FormStep[] = [
         name: 'nextReviewDate',
         label: 'Next review date',
         rules: {
-          required: true,
+          required: 'Pease add a valid date',
```

Comment:
Sorry for being super picky, but some of the errors have a final `.` and some others don't 😅

Reply:
ok will fix